backend/schema/metadataretry.go

```diff
@@ -14,6 +14,7 @@ import (
 )
 
 const (
+	DefaultRetryCount  = 100
```

Comment:
This is the number of retries we attempt if the retry directive does not explicitly give a count.
What's a reasonable default?

Reply:
Great question. Maybe there isn't and we should be explicit, but let's start with this.